tests/unit/test_ops.py

```diff
@@ -1152,32 +1153,32 @@ def test_list_slice(cpu):
     df = DataFrame({"y": [[0, 1, 2, 2, 767], [1, 2, 2, 3], [1, 223, 4]]})
 
     op = ops.ListSlice(0, 2)
-    transformed = op.transform(["y"], df)
+    transformed = op.transform(Columns.from_names("y"), df)
```

Comment:
If we're going to have a `Columns` class, maybe `from_names` could be the `__call__` method? Seems like that would make this part a bit more ergonomic.

Reply:
Oh, I like that! Lemme add that